src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
It would be great to specify what exactly shape or color it could have
![image](https://github.com/mate-academy/ts_figures_classes/assets/96409563/f6e5aa93-d8ab-4253-b560-f16a10c9f03a)